platform/src/main/java/org/stellar/anchor/platform/AnchorPlatformServer.java

```diff
@@ -45,19 +40,15 @@ public static ConfigurableApplicationContext start(
           "management.server.port=8082");
     }
     if (environment != null) {
-      builder.properties(environment);
+      for (String name : environment.keySet()) {
+        System.setProperty(name, String.valueOf(environment.get(name)));
+      }
+      ConfigEnvironment.reset();
```

Comment:
❓ Why are we resetting the ConfigEnvironment?

Reply:
This is to rebuild (renamed) the config environment after the Java system properties are modified.